src/dataframe/dataframe.jl

```diff
@@ -1265,7 +1321,7 @@ end
 """
     push!(df::DataFrame, row::Union{Tuple, AbstractArray})
     push!(df::DataFrame, row::Union{DataFrameRow, NamedTuple, AbstractDict};
-          columns::Symbol=:intersect)
+          cols::Symbol=:equal)
 
 Add in-place one row at the end of `df` taking the values from `row`.
 
```

Comment:
Next 2 lines are only correct for some col types

Reply:
Can you please elaborate on this comment?